src/DiplomaticAction/Alliance/AllianceScoringModel.cs

```diff
@@ -7,21 +7,16 @@ internal sealed class AllianceScoringModel : AbstractScoringModel<AllianceScorin
         public class AllianceScores : IDiplomacyScores
         {
             public int Base => 20;
-
             public int BelowMedianStrength => 50;
-
             public int HasCommonEnemy => 50;
-
             public int ExistingAllianceWithEnemy => -1000;
-
             public int ExistingAllianceWithNeutral => -50;
-
+            public int ExistingAllianceWithAlly => 25;
```

Comment:
Couldn't adding alliance score from alliances start a cascade of unbreakable alliances under certain circumstances? I'm worried that the whole map would end up allied together at some point.

Reply:
That's a good point, I have to reflect on it a bit. I also have some ideas regarding alliance scoring.
Will try to make a bit more complex system and then reopen this request.